cli/src/alluxio.org/cli/processes/common.go

```diff
@@ -0,0 +1,172 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package processes
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+	"path"
+	"strings"
+	"time"
+
+	"golang.org/x/crypto/ssh"
+
+	"alluxio.org/cli/env"
+	"alluxio.org/log"
+)
+
+func getMasters() ([]string, error) {
+	mastersDir := path.Join(env.Env.EnvVar.GetString(env.ConfAlluxioConfDir.EnvVar), "masters")
+	mastersFile, err := os.Open(mastersDir)
+	if err != nil {
+		log.Logger.Errorf("Error reading worker hostnames at %s", mastersDir)
+		return nil, err
+	}
+
+	mastersReader := bufio.NewReader(mastersFile)
+	var mastersList []string
+	lastLine := false
+	for !lastLine {
+		// read lines of the workers file
+		line, err := mastersReader.ReadString('\n')
+		if err != nil {
+			if err == io.EOF {
+				lastLine = true
+			} else {
+				log.Logger.Errorf("Error parsing worker file at this line: %s", line)
+				return nil, err
+			}
+		}
+		// remove notes
+		if strings.Index(line, "#") != -1 {
+			line = line[:strings.Index(line, "#")]
+		}
+		line = strings.TrimSpace(line)
+		if line != "" {
+			mastersList = append(mastersList, line)
+		}
+	}
+	return mastersList, nil
+}
+
+func getWorkers() ([]string, error) {
+	workersDir := path.Join(env.Env.EnvVar.GetString(env.ConfAlluxioConfDir.EnvVar), "workers")
+	workersFile, err := os.Open(workersDir)
+	if err != nil {
+		log.Logger.Errorf("Error reading worker hostnames at %s", workersDir)
+		return nil, err
+	}
+
+	workersReader := bufio.NewReader(workersFile)
+	var workersList []string
+	lastLine := false
+	for !lastLine {
+		// read lines of the workers file
+		line, err := workersReader.ReadString('\n')
+		if err != nil {
+			if err == io.EOF {
+				lastLine = true
+			} else {
+				log.Logger.Errorf("Error parsing worker file at this line: %s", line)
+				return nil, err
+			}
+		}
+		// remove notes
+		if strings.Index(line, "#") != -1 {
+			line = line[:strings.Index(line, "#")]
+		}
+		line = strings.TrimSpace(line)
+		if line != "" {
+			workersList = append(workersList, line)
+		}
+	}
+	return workersList, nil
+}
+
+func getPrivateKey() (ssh.Signer, error) {
+	homePath, err := os.UserHomeDir()
+	if err != nil {
+		log.Logger.Errorf("User home directory not found at %s", homePath)
+		return nil, err
+	}
+	privateKey, err := os.ReadFile(path.Join(homePath, ".ssh", "id_rsa"))
```

Comment:
```suggestion
	privateKeyFile := filepath.Join(homePath, ".ssh", "id_rsa")
	privateKey, err := os.ReadFile(privateKeyFile)
```
use `filepath` package instead of `path` package. store the file path as a variable to reuse for the error messages

a nice to have would be to add a new flag to the command to allow the user to specify a private ssh key. not needed for this PR but something to consider